### PR TITLE
Cross compile `core` with Scala 2.13

### PR DESCRIPTION
**Pull request by joan38**
I am looking into writing a plugin to make sites with Mill and Paradox. My current blocker is that Mill is on 2.13.

Thanks

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -44,6 +44,7 @@ lazy val core = project
   .in(file("core"))
   .settings(
     name := "paradox",
+    crossScalaVersions := Seq(scalaVersion.value, "2.13.2"),
```
Comment (ennru) [inline build.sbt:47]: Isn't cross-compiling the `core` module enough? This will even cross-compile the sbt plugin and the testkit.

Comment (joan38) [reply build.sbt:47]: I thought this would cross compile the core since it's applied on `lazy val core`.
How do you cross compile the core?

Comment (ennru) [reply build.sbt:47]: Ah, sorry. You're right. To the lazy me Github showed it looked like it was `inThisBuild`.
I'll have a closer look...

**Comment by joan38** [comment]
@ennru I removed the one in `inThisBuild` since it will take the on in the `AutoPlugin` anyway.

**Comment by ennru** [comment]
Thank you for your interest in brining Paradox to a broader audience!

**Comment by joan38** [comment]
Thanks!

**Pull request merged**